.circleci/config.yml

```
@@ -298,6 +318,18 @@ commands:
                   echo "Installing volta"
                   msiexec.exe /i "$msi_path" /qn
                   exit $LASTEXITCODE
+            - run:
+                name: Install CMake
+                command: |
+                  $installer_dir = "$Env:TEMP".Path
+                  $download_url = "https://github.com/Kitware/CMake/releases/download/v<< pipeline.parameters.cmake_version >>/cmake-<< pipeline.parameters.cmake_version >>-windows-x86_64.msi"
+                  echo "Downloading CMake"
+                  $msi_path = "$installer_dir\cmake.msi"
+                  (New-Object System.Net.WebClient).DownloadFile("$download_url", "$msi_path")
+                  echo "Installing CMake"
+                  msiexec.exe /i "$msi_path" /qn
+                  [Environment]::SetEnvironmentVariable("Path",  'C:\Program Files\CMake\bin;' + $env:Path, [System.EnvironmentVariableTarget]::User)
+                  exit $LASTEXITCODE
```

```suggestion
                  choco install cmake -y
```

i've ran into issues w/these msi installers being flaky in CCI - choco is maybe a bit more reliable but i don't think you can specify an exact version (though it is currently 3.27.3 in the choco repo which is what you have it pinned to)

---

ok i m gonna do that then, thanks!

---

Ok we're good to go, i m a push the button, thanks!